docs/installation/prepare/jetson-agx-orin-setup.md

```diff
@@ -82,13 +82,24 @@ If this command fails, try using `wget https://storage.googleapis.com/packages.v
 
 ## Serial Communication Protocol Tips
 
+To change the pins that are in use for modes of serial communication, launch <file>jetson-io.py</file> with the following command:
+
+```sh { class="command-line" data-prompt="$"}
+sudo /opt/nvidia/jetson-io/jetson-io.py
+```
+
+In the interactive menu that opens, select **Configure Jetson 40 Pin Header** and **Configure header pins manually** to select and deselect pins to enable use.
+For a Jetson AGX Orin, reference the following:
+
 <!-- prettier-ignore -->
 | Data Sheet ID | GPIO Header Pin | Viam Bus ID | `jetson-io.py` ID | `/dev` Path ID | Notes |
 | ------------- | --------------- | ----------- | ----------------- | ----------- | ----- |
 | I2C_GP2_DAT, I2C_GP2_CLK | 3, 5 | `7` | `i2c2` | `/dev/i2c-2` | |
 | I2C_GP5_DAT, I2C_GP5_CLK | 27, 28 | `1` | `i2c8` | `/dev/i2c-8` | |
```

Comment:
The I2C buses are not configurable through `jetson-io.py`. I suspect they're always-on no matter what. This part will be confusing if it's left as-is.

...OTOH, it's very useful to know that I2C buses 2 and 8 are the relevant ones, and this info probably should go _somewhere._

Reply:
:shrug: It's useful to have it somewhere, I just don't think it goes with the `jetson-io.py` stuff. I guess maybe leave it here with that blank ID and all that good enough? People will figure it out.

Reply:
Upon further reflection I am adding as a note that they're not configurable before pointing to the nvidia documentation underneath the table-- hopefully that will clear up some confusion